AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -218,26 +218,50 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
                 var results: [TranscriptResult] = []
                 rawTranscript.results.forEach { rawResult in
                     var alternatives: [TranscriptAlternative] = []
+                    var languageIdentification: [TranscriptLanguageWithScore] = []
                     rawResult.alternatives.forEach { rawAlternative in
                         var items: [TranscriptItem] = []
+                        var entities: [TranscriptEntity] = []
                         rawAlternative.items.forEach { rawItem in
                             let item = TranscriptItem(type: Converters.Transcript.toTranscriptItemType(type: rawItem.type),
                                                       startTimeMs: rawItem.startTimeMs,
                                                       endTimeMs: rawItem.endTimeMs,
                                                       attendee: Converters.Transcript.toAttendeeInfo(attendeeInfo: rawItem.attendee),
                                                       content: rawItem.content,
-                                                      vocabularyFilterMatch: rawItem.vocabularyFilterMatch)
+                                                      vocabularyFilterMatch: rawItem.vocabularyFilterMatch,
+                                                      stable: rawItem.stable,
+                                                      confidence: rawItem.confidence)
                             items.append(item)
```

Comment:
Same comment applies about merging these two lines all throughout this file (items, entities, alternatives, results, and LanguageWithScores).

Reply:
will be updated.